modules/geo-layers/src/tile-3d-layer/tile-3d-layer.ts

```diff
@@ -105,8 +105,12 @@ export default class Tile3DLayer<DataT = any, ExtraPropsT extends {} = {}> exten
   }
 
   get isLoaded(): boolean {
-    const {tileset3d} = this.state;
-    return tileset3d !== null && tileset3d.isLoaded();
+    return Boolean(
```

Comment:
```suggestion
    return Boolean(this.state?.tileset3d?.isLoaded() && super.isLoaded)
```

Like we do in `TileLayer`